virt/virt-4-13-release-notes.adoc

```diff
@@ -42,6 +42,9 @@ To view the supported guest operating systems for {VirtProductName}, see link:ht
 [id="virt-4-13-new"]
 == New and changed features
 
+// CNV-29445 FIPS not supported in 4.13
+* {product-title} {product-version} is based on {op-system-base-full} 9.2. {op-system-base} 9.2 has not yet been submitted for FIPS validation. Red Hat expects, though cannot commit to a specific timeframe, to obtain FIPS validation for {op-system-base} 9.0 and {op-system-base} 9.2 modules, and later even minor releases of {op-system-base} 9.x. Updates will be available in link:https://access.redhat.com/articles/2918071[Compliance Activities and Government Standards].
```

Comment:
Should this bullet follow closely with what @mjpytlak has done in https://github.com/openshift/openshift-docs/pull/60829/files#diff-5a1bef860ba4f103b12cdc2db661140fa691167a8b50abea180008754482951cR21

Reply:
Oh I see that now. Thanks for adding a second set of eyes for confirmation.

Reply:
@apinnick Thanks for jumping on this so quickly.

Reply:
@apinnick Please hold off on a merge for now. Some new language has come in for layered products from PM in [ocpbugs-14494](https://issues.redhat.com/browse/OCPBUGS-14494)

CC: @peterclauterbach

Reply:
@mjpytlak @opayne1 
I updated the RN text based on Mike's wording.

Reply:
@apinnick Thanks!